CHANGELOG: Fix issue #513 description (#612)
freedomofpress/dangerzone: CHANGELOG.md

- Pass OCR parameters properly in Qubes ([issue #455](https://github.com/freedomofpress/dangerzone/issues/455))
- Fix dark mode support ([issue #550](https://github.com/freedomofpress/dangerzone/issues/550),
  thanks to [@garrettr](https://github.com/garrettr))

> - MacOS/Windows: when accepting checking for updates, the "check for updates" checkbox in the hamburger menu (top-right) would be unchecked, even though it is in fact going to check for updates ([issue #513](https://github.com/freedomofpress/dangerzone/issues/513))

Reviewer: Hm, the wording is a bit complex, I'm afraid. What about?

```suggestion
- MacOS/Windows: Sync "Check for updates" checkbox with the user's choice ([issue #513](https://github.com/freedomofpress/dangerzone/issues/513))
```

Author: done in a89c496.
